jetty-proxy/src/main/java/org/eclipse/jetty/proxy/AbstractProxyServlet.java

```diff
@@ -639,16 +639,20 @@ protected void onProxyResponseFailure(HttpServletRequest clientRequest, HttpServ
         if (_log.isDebugEnabled())
             _log.debug(getRequestId(clientRequest) + " proxying failed", failure);
 
-        int status = failure instanceof TimeoutException ?
-            HttpStatus.GATEWAY_TIMEOUT_504 :
-                HttpStatus.BAD_GATEWAY_502;
+        int status = toStatus(failure);
         int serverStatus = serverResponse == null ? status : serverResponse.getStatus();
         if (expects100Continue(clientRequest) && serverStatus >= HttpStatus.OK_200)
             status = serverStatus;
         sendProxyResponseError(clientRequest, proxyResponse, status);
         
     }
 
+	protected int toStatus(Throwable failure) {
+		return failure instanceof TimeoutException ?
+            HttpStatus.GATEWAY_TIMEOUT_504 :
+                HttpStatus.BAD_GATEWAY_502;
+	}
```

Comment:
1. rename to `proxyResponseStatus()`
2. braces on newline
3. align properly `return` and `:?`